src/pages/home/report/ReportActionCompose/ComposerWithSuggestions.js

```diff
@@ -314,13 +316,16 @@ function ComposerWithSuggestions({
         [suggestionsRef],
     );
 
-    const updateShouldShowSuggestionMenuToFalse = useCallback(() => {
-        if (!suggestionsRef.current) {
+    const hideSuggestionMenu = useCallback(() => {
+        if (!suggestionsRef.current || shouldIgnoreScrollCallback.current) {
+            shouldIgnoreScrollCallback.current = false;
             return;
         }
         suggestionsRef.current.updateShouldShowSuggestionMenuToFalse(false);
     }, [suggestionsRef]);
 
+    const debouncedHideSuggestionMenu = useMemo(() => _.debounce(hideSuggestionMenu, 200, true), [hideSuggestionMenu]);
```

Comment:
We definitely need to explain ourselves out of this `200`, as in Expensify we're (rightfully) skeptical against timeout-based solutions. We need to comment why do this and why 200 ms.